src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -106,6 +106,11 @@ class CombatLogParser extends MainCombatLogParser {
     petrichorLagniappe: PetrichorLagniappe,
     ovydsWinterWrap: OvydsWinterWrap,
   };
+  
+  damageTaken = 0;
```

Comment:
The core parser already provides this in `totalDamageTaken`, so this should be redundant, no?